BlueprintUI/Sources/View Description/UpdateTransition.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UpdateTransition.swift
+//  BlueprintUI
+//
+//  Created by Kyle Bashour on 6/15/20.
+//
+
+import Foundation
+
+public struct UpdateTransition {
+
+    private var _perform: (UIView, @escaping () -> Void) -> Void
+
+    public init(perform: @escaping (UIView, @escaping () -> Void) -> Void) {
```

Comment:
I get exposing the view accessor for the custom case, but it'd be nice to make standard UIView animations easier. Eg, similar to this sort of thing: https://github.com/square/Blueprint/pull/94/files#diff-57cf1fe089b34d04ae53d570db4238f5R44